Added documentation for PackageDescription

CoolDadTx opened the pull request:
## Summary
Added documentation for PackageDescription. Adjusted documentation for Description to convey its dual purpose and when it is used. 

Fixes #8578

CoolDadTx committed:
Added documentation for PackageDescription

Added documentation for PackageDescription. Adjusted documentation for Description to convey its dual purpose and when it is used. #8578

Thraka commented on docs/core/tools/csproj.md:
```diff
@@ -194,9 +194,13 @@ A human-friendly title of the package, typically used in UI displays as on nuget
 A long description of the package for UI display.
 
+### Description
+A long description of the package for UI display. This property is used if `PackageDescription` is not specified. It is also used to specify the generated assembly's description.
```
Thank you for opening this PR! 

After reading through your linked visual studio feedback and the original page, I think we can improve this even further. According to the [AssemblyInfo section](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#assemblyinfo-properties), the **Description** property is really the description of the assembly I think we should describe it that way first and then state that this property will be used as the package description of that **PackageDescription** property is not specified.

CoolDadTx replied:
I agree.

CoolDadTx replied:
Sorry for the delay. I updated the description. 

I noticed that the attributes defined on this page are very inconsistent in how they describe things. Examples:
Authors - A semicolon-separated list
Copyright - Copyright details
PackageOutputPath - Determines the
IncludeSymbols - This Boolean value
Istool  - Specifies whether

I used the terminology that PackageDescription was using but I think this page could stand to have the descriptions reviewed to make them read like they are not copied from documentation elsewhere.

CoolDadTx committed:
Updated Description 

Based upon PR feedback, copied the Description attribute text from main page and modified it to fit the wording of the current page.

Thraka commented on docs/core/tools/csproj.md:
```diff
@@ -194,9 +194,13 @@ A human-friendly title of the package, typically used in UI displays as on nuget
 A long description of the package for UI display.
 
+### Description
+A long description for an assembly. This property is used if `PackageDescription` is not specified.
```
We're close 😄 

As it stands right now, it just sounds like the `Description` property is ignored if the `PackageDescription` property is set. I think we need it made clear that we're saying this property will **also** be used for the generated package if the `PackageDescription` is not specified.

Also, I agree with you about all of the descriptions.

CoolDadTx replied:
OK, how does this sound?

CoolDadTx committed:
Description change

Adjusted text to clarify behavior of Description when PackageDescription is missing.

Thraka commented:
@CoolDadTx PERFECT. Thanks! 😃
